README.md

```diff
@@ -60,7 +60,15 @@ docker run -itd --name OctoBot -p 80:5001 -v $(pwd)/user:/octobot/user -v $(pwd)
 ```
 And then open [http://localhost](http://localhost).
 
-#### [With python sources](https://docs.octobot.online/installation/with-docker)
+#### [With pip](https://docs.octobot.online/installation/with-pip)
+
+In short :
+```
+pip install OctoBot>=0.4.0b14
```

Comment:
actually, if we use `pip install -U OctoBot` we don't need the version

Reply:
Does it work when it is not already installed ?

Reply:
it's 0.3.9 which is installed without version of specified

Reply:
https://pypi.org/project/OctoBot/

Reply:
Right, that's why I said we''ll have to update this line at each update. I don' t know an option to install the latest pre-release version

Reply:
Then I think `OctoBot>=0.4.0b14` is enough not to install 0.3.9, we don't need to update this number alongside the version number in new versions

Reply:
```
~ $ pip3 install -U OctoBot
Collecting OctoBot
  Using cached OctoBot-0.3.9-py2.py3-none-any.whl (682 kB)
```

I agree OctoBot>=0.4.0b14 can do the job. I think we have to stop using pre release version form.

Reply:
right